x-pack/test/observability_ai_assistant_functional/tests/conversations/index.spec.ts

```diff
@@ -287,6 +289,48 @@ export default function ApiTest({ getService, getPageObjects }: FtrProviderConte
                   expect(response.body.conversations[0].messages.length).to.eql(9);
                 });
               });
+
+              describe('and choosing to send feedback', () => {
```

Comment:
can you also add a test where we load an existing conversation, update it, and see if the telemetry that we sent includes the updated conversation?

Reply:
So pre-populate a conversation before the test, open a new conversation in the UI, then load the old conversation, regenerate or add a new prompt, then send feedback and check that the telemetry reflects the updated state (not what was loaded from the API)?

Reply:
yes!

Reply:
Good news: I added a test for it
Bad news: There is a bug in exactly this case

Reply:
Yeah, saw it earlier today :)

Reply:
You could have started by saying there is a bug :D I would have fixed that first and then done the test!

Reply:
Wasn't sure if it was part of my branch 😬